Examples/Scripts/Python/propagation.py

```diff
@@ -49,7 +49,7 @@ def runPropagation(trackingGeometry, field, outputDir, s=None, decorators=[]):
     propagationAlgorithm = acts.examples.PropagationAlgorithm(
         propagatorImpl=propagator,
         level=acts.logging.INFO,
-        sterileLogger=True,
+        sterileLogger=False,
```

Comment:
I've been surprised by this a number of times. However, should we add a python-level printout that states that this is on, and that it significantly increases the runtime of this example?